Automattic/pocket-casts-ios
# Update the API protobuf file

## Description

This runs the `update_proto` script with the latest pocketcasts-api code to generate the latest version of the api.pb.swift file

## To test

1. Clean build the app 
2. Make sure the app builds correctly
3. Make sure the build passes

## Checklist
